keyboards/keebio/iris/keymaps/jontonsoup/keymap.c

```diff
@@ -0,0 +1,130 @@
+#include QMK_KEYBOARD_H
+
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```

Comment:
He highly recommend using an enum, here. 

```suggestion
enum layers {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```